net-misc/gnome-remote-desktop/gnome-remote-desktop-43.0.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
```

Comment:
Often we can remove the Python dependency if we see the removal of something like `meson.add_install_script('meson_post_install.py')`. E.g., in this version bump we have:

````diff
-meson.add_install_script('meson_post_install.py')
+if get_option('man')
+  subdir('man')
+endif
+
+gnome.post_install(
+  glib_compile_schemas: true,
+)
````